var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -24,24 +24,47 @@
 ##############################################################################
 from spack import *
 
-
-class Scr(Package):
+class Scr(CMakePackage):
     """SCR caches checkpoint data in storage on the compute nodes of a
        Linux cluster to provide a fast, scalable checkpoint/restart
        capability for MPI codes"""
 
-    homepage = "https://computation.llnl.gov/project/scr/"
+    homepage = "http://computation.llnl.gov/projects/scalable-checkpoint-restart-for-mpi"
+
+    ## NOTE: scr-v1.1.8 is built with autotools and is not properly build here.
+    ## scr-v1.1.8 will be deprecated with the upcoming release of v1.2.0
+    # url      = "https://github.com/LLNL/scr/releases/download/v1.1.8/scr-1.1.8.tar.gz"
+    # version('1.1.8', '6a0f11ad18e27fcfc00a271ff587b06e')
```

Comment:
There should not be any codes dependent on older versions, and as of now the SCR team does not plan to support versions before 1.2.0 once that version is released.